packages/react/src/index.ts

```diff
@@ -16,6 +16,5 @@ export * from './hooks/wc/useOnSessionExpire.js';
 export * from './hooks/wc/useOnSessionUpdate.js';
 export * from './hooks/wc/useSession.js';
 export * from './provider/PuzzleWalletProvider.js';
-export * from './store.js';
```

Comment:
can remove the file too?

Reply:
No, we need the wallet store, but we shouldn't export it from the index file. I don't want devs to directly access it